package.json

```diff
@@ -35,7 +35,9 @@
     "fix": "gts fix",
     "js-test-application": "mocha bazel-bin/typescript/test/test-application/test-js --timeout 600000",
     "lint": "gts check",
-    "prepack": "npm run compile && cd templates/typescript_gapic && rm -f package.json.njk && mv package.json package.json.njk && cd ../.. && mkdir -p build && cp -rf bazel-bin/typescript templates protos build/",
+    "prepack:cjs": "npm run compile && cd templates/cjs/typescript_gapic && rm -f package.json.njk && mv package.json package.json.njk && cd ../../.. && mkdir -p build && cp -rf bazel-bin/typescript/cjs templates protos build/",
```

Comment:
I'm still getting errors from `cp -rf bazel-bin/typescript/cjs`, where Bazel isn't generating this directory